File: src/main/java/plus/maa/backend/controller/response/user/MaaLoginRsp.java
package plus.maa.backend.controller.response.user;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.Date;

@Data
@AllArgsConstructor
public class MaaLoginRsp {
    private String token;
    private Date validBefore;
    private Date validAfter;
    private String refreshToken;
    private Date refreshTokenValidBefore;
    private Date refreshTokenValidAfter;
    private MaaUserInfo userInfo;
}

File: src/main/java/plus/maa/backend/controller/response/user/MaaLoginRsp.kt
package plus.maa.backend.controller.response.user

import java.time.LocalDateTime

data class MaaLoginRsp(
    val token: String,
    val validBefore: LocalDateTime,
    val validAfter: LocalDateTime,
    val refreshToken: String,
    val refreshTokenValidBefore: LocalDateTime,
    val refreshTokenValidAfter: LocalDateTime,
    val userInfo: MaaUserInfo
)
